### PR TITLE
docs: update old versioned links to unversioned links

### LINKED ISSUE #2000: Update links to expo docs where /versions/latest is no longer required
https://github.com/expo/expo-cli/search?q=%2Fversions%2Flatest&unscoped_q=%2Fversions%2Flatest

### PR DESCRIPTION
**Pull request by byCedric**
Fixes #2000

### CODE REVIEW

File: packages/xdl/src/logs/PackagerLogsStream.ts [hunk 1]
```diff
@@ -416,8 +416,10 @@ export default class PackagerLogsStream {
     const moduleName = match[1];
     const relativePath = path.relative(this._projectRoot, originModulePath);
 
+    // todo: check if we need to re-add this header
+    // https://github.com/expo/expo/commit/f1624cace73fc569402e4304b4b494afa053e046#diff-3bc168b28fe19df4560d1324b96cfa84L103
     const DOCS_PAGE_URL =
```
Comment (byCedric) [inline packages/xdl/src/logs/PackagerLogsStream.ts:421]: I think we might need to change this URL. It's pointing [at a missing header in the FAQ](https://github.com/expo/expo/commit/f1624cace73fc569402e4304b4b494afa053e046#diff-3bc168b28fe19df4560d1324b96cfa84L103).

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ The [Expo CLI GitHub repository](https://github.com/expo/expo-cli) contains the
 - `expo-cli`: Expo CLI is the command line interface for developing, building and sharing Expo apps.
 - `@expo/dev-tools`: the web-based graphical user interface included in Expo CLI for quickly viewing logs, connecting testing devices, deploying updates and more.
 - `xdl`: the Expo development library is a dependency of both the CLI and Dev Tools user interfaces, doing all the heavy lifting behind the scenes.
-- `@expo/schemer`: a library for validating [Expo configuration files](https://docs.expo.io/versions/latest/workflow/configuration).
+- `@expo/schemer`: a library for validating [Expo configuration files](https://docs.expo.io/workflow/configuration).
```
Comment (fson) [inline CONTRIBUTING.md:22]:
```suggestion
- `@expo/schemer`: a library for validating [Expo configuration files](https://docs.expo.io/workflow/configuration/).
```

File: README.md [hunk 3]
```diff
@@ -20,11 +20,11 @@ Expo CLI
     <a aria-label="Circle CI" href="https://circleci.com/gh/expo/expo-cli/tree/master">
     <img alt="Circle CI" src="https://flat.badgen.net/circleci/github/expo/expo-cli?label=Circle%20CI&labelColor=555555&icon=circleci">
   </a>
-  
+
 </p>
 
 <p align="center">
-  <a aria-label="expo documentation" href="https://docs.expo.io/versions/latest/workflow/expo-cli">📚 Read the Documentation</a>
+  <a aria-label="expo documentation" href="https://docs.expo.io/workflow/expo-cli">📚 Read the Documentation</a>
```
Comment (fson) [inline README.md:27]:
```suggestion
  <a aria-label="expo documentation" href="https://docs.expo.io/workflow/expo-cli/">📚 Read the Documentation</a>
```

File: CONTRIBUTING.md (hunk 2 above)
Comment (byCedric) [reply CONTRIBUTING.md:22]: good catch!

**Pull request merged**